GNUmakefile

```diff
@@ -83,11 +81,6 @@ check: test lint
 
 .PHONY: test-compile
 test-compile:
-	@if [ "$(TEST)" = "./..." ]; then \
```

Comment:
TEST by default has a different value: `TEST?=$$(go list ./... | grep -v /integrationtesting)`. Did not find value in the current echos for this specific case.